core/src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -219,6 +220,28 @@ class UserAuthJsonSupport extends JsonSupport {
   implicit val WorkspaceBillingAccountFormat = jsonFormat2(WorkspaceBillingAccount)
 
   implicit val RawlsBillingProjectResponseFormat = jsonFormat7(RawlsBillingProjectResponse)
+
+  implicit val SpendReportingForDateRangeFormat = jsonFormat5(SpendReportingForDateRange)
+
+  implicit val SpendReportingAggregationKeyFormat = jsonFormat1(SpendReportingAggregationKey)
+  implicit val SpendReportingAggregationFormat = jsonFormat2(SpendReportingAggregation)
+  implicit val SpendReportingResultsFormat = jsonFormat2(SpendReportingResults)
+
+
 }
 
 object UserAuthJsonSupport extends UserAuthJsonSupport
+
+// TODO don't dump these case classes here
+case class SpendReportingResults(spendDetails: Seq[SpendReportingAggregation], spendSummary: SpendReportingForDateRange)
+case class SpendReportingAggregation(aggregationKey: SpendReportingAggregationKey, spendData: Seq[SpendReportingForDateRange])
+case class SpendReportingForDateRange(
+                                       cost: String,
+                                       credits: String,
+                                       currency: String,
+                                       startTime: DateTime,
+                                       endTime: DateTime
+                                     )
+
+case class SpendReportingAggregationKey(key: String)
```

Comment:
@aherbst-broad I left this in, but I wasn't sure what you intended the aggregation key to be used for. At the moment it's not really used so I'm inclined to remove it but I wanted to hear from you before doing so

Reply:
So the idea was to express the spend reporting category we are slicing on, ie., "storage", "compute", etc. and have a list of SpendReportingAggregations like this:

```
{
    "spendDetails": [
        {
             "aggreggationKey": "storage",
             "spendData": [ .. time sequence data ... ]
         },
        {
           "aggregationKey": "compute/workflows/whatever",
          "spendData": [ ... time series data ... ]
        }
     ]
```

For the top level aggregation where we're just passing back everything we may want to use a key like "total" instead of empty string.